<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sys_mdbguiClass</class>
 <widget class="QMainWindow" name="sys_mdbguiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sys_mdbgui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../project/res.qrc">
    <normaloff>:/Resources/icon/exe.png</normaloff>:/Resources/icon/exe.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSplitter" name="splitter_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
         <widget class="QSplitter" name="splitter_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>5</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_insert">
              <property name="text">
               <string>插入</string>
              </property>
              <property name="icon">
               <iconset resource="../project/res.qrc">
                <normaloff>:/Resources/icon/insert.png</normaloff>:/Resources/icon/insert.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_del">
              <property name="text">
               <string>删除</string>
              </property>
              <property name="icon">
               <iconset resource="../project/res.qrc">
                <normaloff>:/Resources/icon/delete.bmp</normaloff>:/Resources/icon/delete.bmp</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_save">
              <property name="text">
               <string>保存</string>
              </property>
              <property name="icon">
               <iconset resource="../project/res.qrc">
                <normaloff>:/Resources/icon/save.bmp</normaloff>:/Resources/icon/save.bmp</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>6</horstretch>
              <verstretch>3</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
           </widget>
           <widget class="QTextEdit" name="textEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>6</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>触发记录</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <property name="margin">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit_2">
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="overwriteMode">
              <bool>true</bool>
             </property>
             <property name="acceptRichText">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>查询记录</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="margin">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit_3">
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>操作记录</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="margin">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit_4">
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作(&amp;F)</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="separator"/>
   <addaction name="action_runfile"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset resource="../project/res.qrc">
     <normaloff>:/Resources/icon/exec.png</normaloff>:/Resources/icon/exec.png</iconset>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
   <property name="toolTip">
    <string>执行(F5)</string>
   </property>
  </action>
  <action name="action_runfile">
   <property name="icon">
    <iconset resource="../project/res.qrc">
     <normaloff>:/Resources/icon/runfile.png</normaloff>:/Resources/icon/runfile.png</iconset>
   </property>
   <property name="text">
    <string>运行sql文件</string>
   </property>
   <property name="toolTip">
    <string>运行sql文件</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>断开</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>说明</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>清空注册记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>清空查询记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>清空操作记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../project/res.qrc"/>
 </resources>
 <connections/>
</ui>
